package.json

```diff
@@ -42,16 +42,16 @@
     "babel-loader": "^8.2.5",
     "css-loader": "^6.7.1",
     "html-webpack-plugin": "^5.5.0",
-    "jest": "^29.0.3",
```

Comment:
not a huge deal but curious to hear why you are reverting to older versions of packages like jest and nodemon

Reply:
Should be a mistake, will change to latest version